### ISSUE TITLE
Problem with Integration percy/webdriverio in project "Error: ReferenceError: require is not defined in ES module scope, you can use import instead"

### ISSUE THREAD

**Issue by mishko-misha** [webdriverio/appium-boilerplate#525]
**Environment (please complete the following information):**
 - Node.js version: v 20.11.1
 - NPM version: 10.8.1
 - Platform: Android
 - Platform version: 13.0
 - Emulator/Simulator/Real Device: Emulator from Browserstack
 - TypeScript: 5.4.5

**Describe the bug**
1. npm install --save-dev @percy/cli @percy/webdriverio
2. Added require and await percySnapshot (see screen) in test
3. Run test

**Actual result:** I get failed with error "Error: ReferenceError: require is not defined in ES module scope, you can use import instead". (check **Logs when I use require.txt**)
[Logs when I use require.txt](https://github.com/user-attachments/files/18238509/Logs.when.I.use.require.txt)
![My test with require](https://github.com/user-attachments/assets/c2bd21fa-05cd-4e3a-ab80-8c8b00ceb54a)

4. Replaced **const percySnapshot = require('@percy/webdriverio');** to **import percySnapshot from '@percy/webdriverio';**

**Actual result:** I get failed (check **Logs when I use import.txt**)
[Logs when I use import.txt](https://github.com/user-attachments/files/18238508/Logs.when.I.use.import.txt)
![My test with import - error](https://github.com/user-attachments/assets/3d6a55d8-d9ac-49e8-b37d-0126ba777f74)
**From Percy:** [37950736_cli.json](https://github.com/user-attachments/files/18238512/37950736_cli.json)

**FYI:** I opened @percy/webdriverio and saw @percy/core **The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("@percy/core")' call instead.**
![Percy-Core](https://github.com/user-attachments/assets/c8428ad0-72b0-4a11-bed2-98df7fe3d910)

**Additional context**
Settings:
![packagejson](https://github.com/user-attachments/assets/87e35f48-bd5c-437e-bf68-314334c60e67)
![tsconfig](https://github.com/user-attachments/assets/17fa5539-e554-4faa-bd68-9022411bc5a8)

Could you give me advice how to fix my problem?

**Comment by christian-bromann** [comment]
Can you provide a minimal reproduction case?

**Comment by mishko-misha** [comment]
> Can you provide a minimal reproduction case?

1. Clone project
2. Install necessary dependencies ( you can see screenshots above)
3. Created android config
4. Create page object class and test (you can see screenshots above)
5. Run test

Actual result: test works fine and I can check result in browserstack but get error Error: ReferenceError: require is not defined in ES module scope, you can use import instead (also described above)

**Comment by christian-bromann** [comment]
@mishko-misha can you share a GitHub project that has executed the steps so that maintainers only need to clone, install deps and run a command?

**Comment by mishko-misha** [comment]
> @mishko-misha can you share a GitHub project that has executed the steps so that maintainers only need to clone, install deps and run a command?

yes, I can but without my access to browserstack, percy, app.apk. If you agree, I will send. agree?

**Comment by christian-bromann** [comment]
> yes, I can but without my access to browserstack, percy and app.apk. If you agree, I will send.

Ideally, reduce everything unnecessary to reproduce the problem. This error is not related to BrowserStack or which capabilities you are using. The simpler the reproduction case the better.

**Comment by mishko-misha** [comment]
@christian-bromann my project https://github.com/mishko-misha/Mobile-project/tree/main

**Comment by mishko-misha** [comment]
@christian-bromann do you have any news?

**Comment by mpolunin** [comment]
> [@christian-bromann](https://github.com/christian-bromann) do you have any news?

@christian-bromann  I am still encountering this problem

**Comment by mpolunin** [comment]
@christian-bromann I am still encountering this problem
